scanners/kubeaudit/examples/juice-shop/README.md

```diff
@@ -0,0 +1,31 @@
+<!--
+SPDX-FileCopyrightText: the secureCodeBox authors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+In this example we execute an kubeaudit scan against the intentional vulnerable [juice-shop](https://github.com/juice-shop/juice-shop)
+
+#### Initialize juice-shop in cluster
+
+Before executing the scan, make sure to setup juice-shop
+
+```bash
+helm upgrade --install juice-shop secureCodeBox/juice-shop --wait
+```
+
+After that you can execute the scan in this directory:
+```bash
+kubectl apply -f scan.yaml
+```
+
+#### Troubleshooting:

```

Comment:
This seems more like an example workflow, than a troubleshooting section. Either remove the header, or reformat the section to fit it. Your call :)